world/zoneserver.cpp

```diff
@@ -889,6 +889,11 @@ void ZoneServer::HandleMessage(uint16 opcode, const EQ::Net::Packet &p) {
 		zoneserver_list.SendPacket(pack);
 		break;
 	}
+	case ServerOP_ReloadLevelEXPMods:
```

Comment:
Whenever these are all merged, all of these reloads could be stacked ontop of eachother as cases
eg

```cpp
case1:
case2:
case3:
    zoneserver_list.SendPacket(pack);
```

Reply:
Yeah, absolutely. 👍